samples: bluetooth: fast_pair: add note about device type in readme (#15255)
nrfconnect/sdk-nrf: samples/bluetooth/fast_pair/input_device/README.rst

Before you can use your device as a Fast Pair Provider, the device model must be registered with Google.
This is required to obtain Model ID and Anti-Spoofing Private Key.
You can register your own device or use the debug Model ID and Anti-Spoofing Public/Private Key pair obtained by Nordic for development purposes.

.. note::

>    To support the input device use case, you must select :guilabel:`Input Device` option in in the **Device Type** list when registering your device model.

Reviewer: ```suggestion
   To support the input device use case, you must select :guilabel:`Input Device` option in the **Device Type** list when registering your device model.
```

   :start-after: nrf_desktop_fastpair_important_start
   :end-before: nrf_desktop_fastpair_important_end

> .. important::

Reviewer: We could move this "admonitions" after line no: 74 because now we have 3 admonitions one after the other in the documentation.

Author: I decided to move it before the part where we describe the default model as the note is only relevant when the user decides to use their device model instead of the default Nordic model

> .. important::
>    To support the input device use case, you must select the ``input device`` option in the **Device Type** field during device model registration.

Reviewer 2: ```suggestion
.. note::
   To support the input device use case, you must select the **input device** option in the **Device Type** field when registering your device model.
```
I'd suggest to change this into a regular note

Author: ok

>    To support the input device use case, you must select the ``input device`` option in the **Device Type** field during device model registration.

Reviewer: Is "input device" a clickable element? If so, it should be represented as ":guilabel:`input device`"

Author: Yes, accessed via dropdown list:

![Screenshot from 2024-05-10 11-43-07](https://github.com/nrfconnect/sdk-nrf/assets/17042049/c03b8d7c-1291-44d3-9d74-d033cc41fa67)

Author: discussed offline: 

```
you must select :guilabel:`Input Device` option in in the **Device Type** list .....
```

>    To support the input device use case, you must select the ``input device`` option in the **Device Type** field during device model registration.

Reviewer 3: `Input Device` (to match label from Nearby Console)?
